src/components/_classes/input/Input.js

```diff
@@ -26,8 +26,8 @@ export default class Input extends Multivalue {
       lang: this.options.language
     };
 
-    if (this.options.attachMode === 'builder' || this.options.building) {
-      attr.autocomplete = 'off';
+    if (this.options.attachMode === 'builder' || this.options.building || _.get(this.root, 'form.settings.disableAutocomplete')) {
+      attr.autocomplete = this.component.type === 'password'? 'new-password': 'off';
```

Comment:
All Password component settings should be in the Password component class. 
I would prefer to add new getter (e.g. 'autocompleteDisableAttrName') that will return 'off' in the Input class and some other value in the Password component class.

Reply:
added getter